python/examples/llama-index/backend/app/engine/loaders/__init__.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+import yaml
+
+from app.engine.loaders.file import FileLoaderConfig, get_file_documents
+
+logger = logging.getLogger(__name__)
+
+
+def load_configs():
+    with open("config/loaders.yaml") as f:
+        configs = yaml.safe_load(f)
+    return configs
+
+
+def get_documents():
+    documents = []
+    config = load_configs()
+    for loader_type, loader_config in config.items():
+        logger.info(f"Loading documents from loader: {loader_type}, config: {loader_config}")
+        match loader_type:
+            case "file":
+                document = get_file_documents(FileLoaderConfig(**loader_config))
+            case _:
+                raise ValueError(f"Invalid loader type: {loader_type}")
```

Comment:
This Python syntax was introduced in Python 3.10. If we haven't already, let's put a warning in the README that this project requires Python 3.10 or later to run from source (not to run with Docker compose, though).

Reply:
Yes, you're right, I had to setup an env with py ^3.11 as specified in .toml file.